src/BinaryParsers/BinaryParsers.csproj

```diff
@@ -43,4 +43,14 @@
     </EmbeddedResource>
   </ItemGroup>
 
-</Project>
+  <PropertyGroup>
+    <PreBuildEvent Condition="$(OS) == 'Windows_NT'">
+      echo Copying file from "$(MSBuildProjectDirectory)\..\..\refs\Dia2Lib.dll" to "$(MSBuildProjectDirectory)\..\..\src\packages\microsoft.diagnostics.tracing.traceevent\3.1.3\lib\netstandard2.0\"
+      copy "$(MSBuildProjectDirectory)\..\..\refs\Dia2Lib.dll" "$(MSBuildProjectDirectory)\..\..\src\packages\microsoft.diagnostics.tracing.traceevent\3.1.3\lib\netstandard2.0\"
```

Comment:
This is to replace with our version of Dia2Lib.dll that works with our code.

Reply:
This is good point if it will break the generation of events.
I am not familiar about it,
I see the code we are doing check like:
`if (!string.IsNullOrEmpty(globalContext.EventsFilePath))`
to enable generation of events, and the setting of `Context.EventsFilePath` is:
`context.EventsFilePath = Environment.GetEnvironmentVariable("SPMI_ETW") ?? options.EventsFilePath ?? context.EventsFilePath;`
Is this only enabled in SPMI?